manage.py

```diff
@@ -1,24 +1,23 @@
 #!/usr/bin/env python
 import os
 
-# Edit this if necessary or override the variable in your environment.
-os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'fjord.settings')
-
 from fjord import manage_utils
 
-manage_utils.setup_environ(__file__)
-
-from fjord.base import monkeypatches
-monkeypatches.patch()
 
+if __name__ == "__main__":
+    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'fjord.settings')
 
-SKIP_CHECK = os.environ.get('SKIP_CHECK', '0')
-USING_VENDOR = os.environ.get('USING_VENDOR', '0')
+    print 'Using %s' % os.environ['DJANGO_SETTINGS_MODULE']
```

Comment:
Scratch that--I only see it once now. I'm not sure why you'd be seeing it twice. Maybe tweak it so that it prints the sys.argv, too, so then you can see whether manage.py is being executed twice with different args?